charts/piraeus/templates/operator-satelliteset.yaml

```diff
@@ -4,6 +4,16 @@ kind: LinstorSatelliteSet
 metadata:
   name: {{ template "operator.fullname" . }}-ns
   namespace: {{ .Release.Namespace }}
+  labels:
+    name: {{ template "operator.fullname" . }}-satellite-set
+{{- with .Values.operator.satelliteSet.customLabels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+  annotations:
+    name: {{ template "operator.fullname" . }}-satellite-set
```

Comment:
and here again: use the same name as in the `name` field, and remove from annotations